src/Components/Navbar.css

```diff
@@ -0,0 +1,60 @@
+.nav-bar {
+  display: flex;
+  align-items: end;
+  width: 87.5rem;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+}
+
+.book-heading {
+  height: 2.313rem;
+  margin: 0.375rem 2.938rem 0.125rem 0;
+
+  /*   font-family: Montserrat;
+ */
+  font-size: 1.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.nav-items {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 3rem;
+  list-style: none;
+}
+
+.books-link {
+  align-items: center;
+
+  /*   font-family: Montserrat;
+ */
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 1.9px;
+  color: #121212;
+  text-decoration: none;
+}
```

Comment:
[OPTIONAL] One of the Bad CSS Practices is using the Same Rule Repetitively. I noticed that you use repetitive rules for each element. If you want to create a rule for multiple elements, use a class attribute instead.
For instance, instead of repeating font params (font-weight, font-style...) for each class:
```css
.book-heading {
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
}

.books-link {
  font-stretch: normal;
  font-style: normal;
  line-height: normal;
}
```
Use a class instead. Ex:
```css
.font-params {
   font-stretch: normal;
   font-style: normal;
   line-height: normal;
}
```